pkg/packagekit/updates.jsx

```diff
@@ -1472,23 +1473,25 @@ class OsUpdates extends React.Component {
             });
 
             return (
-                <>
+                <div className='update-error-main-view'>
                     <EmptyStatePanel title={ STATE_HEADINGS[this.state.state] }
                                      icon={ ExclamationCircleIcon }
                                      paragraph={
-                                         <TextContent>
-                                             <Text component={TextVariants.p}>
-                                                 {this.state.errorMessages
-                                                         .filter((m, index) => index == 0 || m != this.state.errorMessages[index - 1])
-                                                         .map(m => <span key={m}>{m}</span>)}
-                                             </Text>
-                                             <Text component={TextVariants.p}>
-                                                 {_("Please reload the page after resolving the issue.")}
-                                             </Text>
-                                         </TextContent>
+                                         <Text component={TextVariants.p}>
+                                             {_("Please reload the page after resolving the issue.")}
+                                         </Text>
```

Comment:
This part should probably still come after the error messages.

The flow of information is:

1. Error title
2. Error details
3. How to resolve

Switching it around with error title, resolution, and then error details is a bit odd.

Reply:
Additionally, we could change it to "Please resolve the issue and reload this page." This would put it in a more logical order.

(This isn't related to this PR, but if you're touching that part of the code anyway, we could improve it too.)

Reply:
Yes. I have changed the How to resolve.